composer.json

```diff
@@ -19,5 +19,8 @@
         "react/event-loop": "0.3.*|0.4.*",
         "react/promise": "~1.0|~2.0",
         "ext-soap": "*"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^4.8"
```

Comment:
This is *debatable*… :) As such, I'd like to leave this out for now and discuss this in a separate ticket if desired.